engine/dlib/src/dlib/log.cpp

```diff
@@ -303,11 +306,16 @@ static void DoLogPlatform(LogSeverity severity, const char* output, int output_l
 #endif
 
 #ifdef __EMSCRIPTEN__
+
         //Emscripten maps stderr to console.error and stdout to console.log.
         if (severity == LOG_SEVERITY_ERROR || severity == LOG_SEVERITY_FATAL){
-            fwrite(output, 1, output_len, stderr);
+            EM_ASM_({
+                Module.printErr(UTF8ToString($0));
+            }, output);
         } else {
-            fwrite(output, 1, output_len, stdout);
+            EM_ASM_({
+                Module.print(UTF8ToString($0));
+            }, output);
```

Comment:
This approach allows to print full message as one console entity:
![CleanShot 2024-07-02 at 18 53 36@2x](https://github.com/defold/defold/assets/2209596/ec09d270-b988-40e0-9a2c-6aeca72beed4)